src-ui/src/app/components/common/confirm-dialog/confirm-dialog.component.ts

```diff
@@ -34,6 +35,8 @@ export class ConfirmDialogComponent implements OnInit {
   confirmButtonEnabled = true
   seconds = 0
 
+  subject: Subject<boolean>
```

Comment:
pls do not use technical names for members. This makes it very hard to get the intention of what will happen here.